### PR TITLE
Enables lighting cigarettes with Wands of Fireball

### PR DESCRIPTION
**Pull request by SamHPurp**
image

:cl: Purpose2
add: You can now light cigarettes with Wands of Fireball, at the cost of your eyebrows.
/ :cl:

### CODE REVIEW

File: code/game/objects/items/weapons/cigs.dm [hunk 1]
```diff
@@ -95,6 +95,17 @@ LIGHTERS ARE IN LIGHTERS.DM
 	else if(istype(W, /obj/item/device/assembly/igniter))
 		light("<span class='notice'>[user] fiddles with [W], and manages to light their [name].</span>")
 
+	else if(istype(W, /obj/item/weapon/gun/magic/wand/fireball))
+		var/obj/item/weapon/gun/magic/wand/fireball/F = W
+		if(F.charges)
+			if(prob(50))
+				light("<span class='notice'>Trying painfully hard to be cool, [user] manages to light their [name] with [W], singing their eyebrows.</span>")
+			else
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/cigs.dm]: This should never happen if you're a wizard.

They're too....magically-ish for this to happen.

File: code/game/objects/items/weapons/cigs.dm [hunk 2]
```diff
@@ -95,6 +95,17 @@ LIGHTERS ARE IN LIGHTERS.DM
 	else if(istype(W, /obj/item/device/assembly/igniter))
 		light("<span class='notice'>[user] fiddles with [W], and manages to light their [name].</span>")
 
+	else if(istype(W, /obj/item/weapon/gun/magic/wand/fireball))
+		var/obj/item/weapon/gun/magic/wand/fireball/F = W
+		if(F.charges)
+			if(prob(50))
+				light("<span class='notice'>Trying painfully hard to be cool, [user] manages to light their [name] with [W], singing their eyebrows.</span>")
```
Comment (FlattestGuitar) [inline code/game/objects/items/weapons/cigs.dm]: Correct me if I'm wrong, but I'm pretty sure this message should be the most badass one you could possibly witness in this video game.

Comment (SamHPurp) [inline code/game/objects/items/weapons/cigs.dm]: I was unsure which side of the line to walk, mocking edgelord, or HOLY FUCK WHOAAAAAAAAA.

**Comment by granodd** [comment]
"[user] lights their [name] in a casual manner with their [w]. You're unsure if they're just so cool that they don't care or are completely insane to consider using the instrument of fiery, explosive death as an alternative to a lighter."

I like this. It would give me yet another thing to light Jeriko's cigarettes with if he manages to steal one or is a wizard himself.

File: code/game/objects/items/weapons/cigs.dm (hunk 1 above)
Comment (SamHPurp) [inline code/game/objects/items/weapons/cigs.dm]: Removes a lot of the risk and amusement, but I've added the check for muggledom.

File: code/game/objects/items/weapons/cigs.dm (hunk 2 above)
Comment (Anticept) [inline code/game/objects/items/weapons/cigs.dm]: You managed to do both.

**Pull request merged**